webapp/channels/src/components/common/hooks/useExpandOverageUsersCheck.ts

```diff
@@ -1,39 +1,21 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {useCallback} from 'react';
 import {defineMessage} from 'react-intl';
 import {useSelector} from 'react-redux';
 
-import {trackEvent} from 'actions/telemetry_actions.jsx';
 import {getExpandSeatsLink} from 'selectors/cloud';
 
-type UseExpandOverageUsersCheckArgs = {
-    isWarningState: boolean;
-    banner: 'global banner' | 'invite modal';
-}
-
 const cta = defineMessage({
     id: 'licensingPage.overageUsersBanner.cta',
     defaultMessage: 'Contact Sales',
 });
 
-export const useExpandOverageUsersCheck = ({
-    isWarningState,
-    banner,
-}: UseExpandOverageUsersCheckArgs) => {
+export const useExpandOverageUsersCheck = () => {
     const expandableLink = useSelector(getExpandSeatsLink);
```

Comment:
Similarly, `expandableLink` isn't actually used any more either, but that's also a pre-existing issue